src/_settings.scss

```diff
@@ -0,0 +1,763 @@
+ * of the numbers that the weights correspond to, plus it ensures that your
+ * project's typography is applied consistency. You can use these settings for
+ * none webfonts i.e. system fonts but typically you'll only need to apply one
+ * weight which is **bold** and in that case using `font-weight: bold;` is
+ * fine.
+ */
+
+$scally-g-font-weight-normal: 400 !default;
+
+$scally-g-font-weight-thin: 100 !default;
+
+$scally-g-font-weight-extra-light: 200 !default;
+
+$scally-g-font-weight-light: 300 !default;
+
+$scally-g-font-weight-medium: 500 !default;
+
+$scally-g-font-weight-semi-bold: 600 !default;
+
+$scally-g-font-weight-bold: 700 !default;
+
+$scally-g-font-weight-extra-bold: 800 !default;
+
+$scally-g-font-weight-black: 900 !default;
+
+// Heading weight
+$scally-g-font-weight-heading: $scally-g-font-weight-bold !default;
+
+
+/**
+ * Maximum line length using the `ch` unit.
+ */
+
+$scally-g-max-line-length: 600ch !default;
+
+
+
+
+/* 3. Spacing (Global)
+   ========================================================================= */
+
+/**
+ * The base spacing unit uses the base line height to achieve a vertical
+ * rhythm.
+ */
+
+$scally-g-spacing-base: $scally-g-line-height !default;
+
+// Decrease
+$scally-g-spacing-small: floor($scally-g-spacing-base / 1.5) !default;
+
+$scally-g-spacing-x-small: floor($scally-g-spacing-base / 2) !default;
+
+$scally-g-spacing-xx-small: floor($scally-g-spacing-base / 3) !default;
+
+$scally-g-spacing-xxx-small: floor($scally-g-spacing-base / 4) !default;
+
+$scally-g-spacing-xxxx-small: floor($scally-g-spacing-base / 6) !default;
+
+// Increase
+$scally-g-spacing-large: ceil($scally-g-spacing-base * 1.3) !default;
+
+$scally-g-spacing-x-large: ceil($scally-g-spacing-base * 1.5) !default;
+
+$scally-g-spacing-xx-large: ceil($scally-g-spacing-base * 1.7) !default;
+
+$scally-g-spacing-xxx-large: ceil($scally-g-spacing-base * 2) !default;
+
+$scally-g-spacing-xxxx-large: ceil($scally-g-spacing-base * 2.2) !default;
+
+
+
+
+/* 4. Breakpoints (Global)
+   ========================================================================= */
+
+/**
+ * The breakdown of the breakpoints is based off this chart:
+ * http://static.lukew.com/unified_device_design.png
+ *
+ * Here's the thinking:
+ *
+ * - Palm..........smart-phones and feature phones—anything that can fit into
+ *                 the palm of a hand.
+ * - Lap...........typically tablets, notebooks and laptops—anything that is
+ *                 portable that doesn't include phones.
+ * - Lap small.....smaller tablets/notebooks and e-readers e.g. Kindle.
+ * - Lap large.....larger tablets and laptops.
+ * - Desk..........desktop computers, TVs, etc.—anything that isn't portable.
+ * - Desk large....larger desktop computer monitors, TVs, and who knows what
+ *                 else?
+ *
+ * The breakpoints:
+ *
+ * - Palm: 0–640px
+ * - Lap: 641px +
+ * - Lap small: 768px +
```

Comment:
None of them are specifically created to be a range, would it make sense to just switch the order to be `Lap small: 641` and `Lap: 768`. This change would make it non-compatible with v2 and make upgrading a massive pain though.